src/applications/pre-need/tests/definitions/address.unit.spec.jsx

```diff
@@ -34,7 +34,7 @@ describe('Pre-need definition address', () => {
     // country is USA and there is no blank option
     expect(selects.first().props().value).to.equal('USA');
     expect(selects.first().find('option').everyWhere(n => !!n.props().value)).to.be.true;
-  }).timeout(4000);
```

Comment:
Oh interesting. Didn't know this was a thing. Seems like removing this changes the way the test works, doesn't it?